### PR TITLE
Revising CI Status functionality

### PR DESCRIPTION
**Pull request by danmacpherson**
This includes the following:

- dashboard v2
- Patternfly-generated shields
- Patternfly layouts
- Improved filtering on the dashboard UI
- Fixing Internal CI page

image
image
image

### CODE REVIEW

**Review by abhatt-rh — COMMENTED** [review]
Thank you @danmacpherson for this PR. Just had a ques, but the rest looks great to me.
The neat links on the Pattern-specific pages for bug reporting, help and such and the  sorting options on the CI page are quite nifty.

File: content/ci/internal.md
[the remark this answers is not in the document]
Comment (danmacpherson) [reply content/ci/internal.md:1]: Not sure which other link you mean? Can you elaborate a bit more?

[the remark this answers is not in the document]
Comment (danmacpherson) [reply content/ci/internal.md:1]: Just discussed with Andrew. Removing the link on the index page. Only people who know the page exists can access the page.

**Comment by beekhof** [comment]
/lgtm 
ship it!

**Pull request merged**